config/data_scrubber_test.go

```diff
@@ -41,14 +43,14 @@ func setupDataScrubberWildCard(t *testing.T) *DataScrubber {
 
 	expectedPatterns := make([]string, 0, len(defaultSensitiveWords))
 	for _, word := range append(defaultSensitiveWords) {
```

Comment:
The setup functions already use the public constructor on 
```
scrubber := NewDefaultDataScrubber()
scrubber.AddCustomSensitiveWords(wildcards)
```

I created the regex expressions on the test just to verify if these two methods above created the expected regex patterns on the DataScrubber object for the default and custom words. 
The idea was that the private setup functions test the internals of dataScrubber and the other public Test functions test the interface.

Do you think that it's better to get rid off the regex pattern asserts since they test the internals?